docs/content/community/contributing.mdx

```diff
@@ -9,6 +9,53 @@ We love to see our community members get involved! If you are planning to contri
 
 ## Environment Setup
 
+### Apple M1 specific setup
+
+If you have a Mac with an M1 chip, you will need to set up a separate environment to emulate an x86 architecture before following the remaining setup instructions.
```

Comment:
Maybe explicitly call out that, otherwise you can skip this section. Also, are we able to link to something that helps users discover whether they have an M1 chip?